Corona-Warn-App/src/main/res/layout/fragment_interoperability_configuration.xml

```diff
@@ -23,35 +23,24 @@
             app:layout_constraintStart_toStartOf="parent"
             app:layout_constraintTop_toTopOf="parent" />
 
-        <androidx.constraintlayout.widget.ConstraintLayout
-            android:id="@+id/interoperability_configuration_country_list_container"
+        <include
```

Comment:
The TextView now touches the bottom screen end.
I think you might have forgot to readd `android:layout_marginBottom` when refactoring?

Reply:
Actually, I did not forget but as it was reused adding a padding to the end of the layout messed up the onboarding. I refactored to solve this issue. The preview looks almost like the actual screen now.